recipes/tinyspline/all/patches/0.5.0-fix-parson.patch

```diff
@@ -0,0 +1,168 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -7,6 +7,7 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+ option(TINYSPLINE_BUILD_EXAMPLES "Build TinySpline examples." ON)
+ option(TINYSPLINE_BUILD_TESTS "Build TinySpline tests." ON)
+ option(TINYSPLINE_BUILD_DOCS "Build TinySpline documentation." ON)
++option(TINYSPLINE_JSON_SUPPORT "Build TinySpline with JSON support." ON)
+ 
+ if(NOT DEFINED TINYSPLINE_OUTPUT_DIRECTORY)
+   set(TINYSPLINE_OUTPUT_DIRECTORY
+--- src/CMakeLists.txt
++++ src/CMakeLists.txt
+@@ -884,7 +884,6 @@ set(TINYSPLINE_CXX_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
+ # TINYSPLINE_C_SOURCE_FILES
+ list(APPEND TINYSPLINE_C_SOURCE_FILES
+      "${CMAKE_CURRENT_SOURCE_DIR}/tinyspline.c"
+-     "${CMAKE_CURRENT_SOURCE_DIR}/parson.c"
```

Comment:
But you have to append this compilation unit again when `TINYSPLINE_JSON_SUPPORT` is ON, and I don't see this logic in your patch.